packages/rocketchat-lib/server/functions/createRoom.js

```diff
@@ -82,7 +82,9 @@ RocketChat.createRoom = function(type, name, owner, members, readOnly, extraData
 		RocketChat.models.Subscriptions.createWithRoomAndUser(room, member, extra);
 	}
 
-	RocketChat.authz.addUserRoles(owner._id, ['owner'], room._id);
+	if (!extraData.automatic) {
```

Comment:
No I don't think there should be an owner, because else the owner(which will be the first user added to that channel) could change setting like channel Name or room topic.